learn-ocaml.el

```diff
@@ -118,6 +121,12 @@ Call `get-buffer-create' if need be, to ensure it is a live buffer."
              (current-time-string)
              " ---------------------\n")))
 
+(defun escape-secret (secret)
+  "Add escape to the secret when it's empty"
+  (if-let (secret-option (string= secret ""))
+      "\"\""
+      secret))
+
```

Comment:
@LouisAyroles 
If this function had to be kept, it should be renamed to `learn-ocaml-escape-secret`.

However, in line with my previous review comment https://github.com/pfitaxel/learn-ocaml.el/pull/20#discussion_r663887665 :

I believe that this function `escape-secret` can be removed, as if you use `learn-ocaml-make-process-wrapper` or so, the `" \"\" "` fragile hack is not needed anymore! :)

However, maybe to be on the safe side, you could replace the occurrences of `(escape-secret secret)` with `(or secret "")` so that, if `secret` is non-nil (a proper string), it is taken into account, otherwise, if `secret` is nil, it will pass `""` to the list of arguments to `make-process`…